src/pages/atomicdex/api/v20-dev/lightning/index.mdx

```diff
@@ -0,0 +1,144 @@
+- List open lightning channels matching a filter [lightning::channels::list_open_channels_by_filter](/atomicdex/api/v20-dev/lightning/channels#list_open_channels_by_filter)
+- List closed lightning channels matching a filter [lightning::channels::list_closed_channels_by_filter](/atomicdex/api/v20-dev/lightning/channels#list_closed_channels_by_filter)
+
+
+## Lightning Network Payments Methods
+
+- Generate lightning invoices with [lightning::payments::generate_invoice](/atomicdex/api/v20-dev/lightning/payments#generate_invoice)
+- Send lightning payments with [lightning::payments::send_payment](/atomicdex/api/v20-dev/lightning/payments#send_payment)
+- Get details about a lightning payment with [lightning::payments::get_payment_details](/atomicdex/api/v20-dev/lightning/payments#get_payment_details)
+- Get a filtered list of lightning payments with [lightning::payments::list_payments_by_filter](/atomicdex/api/v20-dev/lightning/payments#list_payments_by_filter)
+
+
+## Lightning Network Flowchart
+
+Once you:
+- Enable lightning with [task::enable_lightning::init](/atomicdex/api/v20-dev/lightning/activation#init)
+- Connect to a lightning node with [lightning::nodes::connect_to_node](/atomicdex/api/v20-dev/lightning/nodes#connect_to_node)
+- Open a lightning channel [lightning::channels::open_channel](/atomicdex/api/v20-dev/lightning/channels#open_channel)
+
+You can get an invoice for a coffee from https://starblocks.acinq.co!
+
+Then you can:
+- Pay the invoice with [lightning::payments::send_payment](/atomicdex/api/v20-dev/lightning/payments#send_payment)
+- View details about the payment with [lightning::payments::get_payment_details](/atomicdex/api/v20-dev/lightning/payments#get_payment_details)
+- List your payments with [lightning::payments::list_payments_by_filter](/atomicdex/api/v20-dev/lightning/payments#list_payments_by_filter)
+
+Follow the flowchart below to visualize the process:
+
+<OptimizedImage src={lightningflowchart} classNaming="w-full" alt="Lighning Network Flowchart" />
+
+
+## Lightning Network Coins File Configuration
+
+<Note>
+Some configurations are set per coin, and some are set per channel. The [counterparty_channel_config_limits](/atomicdex/api/v20#counterparty-channel-options) param must be set in the `coins` configuration file, an aplies to all channels opened by counterparty nodes. The [our_channels_config](/atomicdex/api/v20#lightning-channel-config) and [channel_options](/atomicdex/api/v20#lightning-channel-options) parameters are set per channel. These can be defined in the `coins` configuration file to act as the default for all opened channels, and optionally overwritten or updated using the [open_channel](/atomicdex/api/v20-dev/lightning/channels#open_channel) or [update_channel](/atomicdex/api/v20-dev/lightning/channels#update_channel) methods.
+</Note>
+
+
+### Lightning Coin Config Parameters
```

Comment:
You missed 2 parameters in the lightning config, these are:

`accept_inbound_channels` optional `bool`, defaults to `true`
If this is set to false, we do not accept inbound requests to open a new channel.

`accept_forwards_to_priv_channels` optional `bool`, defaults to `false`.
When set to false, any HTLCs which were to be forwarded over private channels will be rejected. This prevents us from taking on HTLC-forwarding risk when we intend to run as a node which is not online reliably (e.g. GUIs/user wallets), usually private channels are used for non-routing purposes which is why this defaults to false and `channel_options::announced_channel` should be false for GUIs/wallets.

Reply:
Did you mean `our_channels_config::announced_channel` ?

Reply:
updates applied